apps/emqx/src/emqx_authentication_config.erl

```diff
@@ -52,7 +52,6 @@
 -type update_request() ::
     {create_authenticator, chain_name(), map()}
     | {delete_authenticator, chain_name(), authenticator_id()}
-    | {delete_authenticators, chain_name()}
```

Comment:
Just confirming: after this change, the user will have to delete each authenticator individually to clear the chain?

Reply:
Yes. After all authenticators are removed, the chain is removed too.

Reply:
Just curiosity: any particular reason to remove this API?